includes/class-amp-theme-support.php

```diff
@@ -516,7 +536,7 @@ public static function add_amp_comment_form_templates() {
 		?>
 		<div submit-success>
 			<template type="amp-mustache">
-				<?php esc_html_e( 'Your comment has been posted, but may be subject to moderation.', 'amp' ); ?>
+				{{{data}}}
```

Comment:
The error handler sends back an object with an `error` key. I think this should be similarly looking for an object returned with just a `message` key.